tests/network/test_availability.py

```diff
@@ -0,0 +1,56 @@
+import maya
```

Comment:
We could also verify `ursula._availability_sensor.status()` as part of the tests. See my code suggestion for the bug in the `status()` method

Reply:
:heavy_check_mark: